roles/openshift_metrics/tasks/generate_certificates.yaml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
@abutcher any suggestions on using the openshift CA with OpenSSL here instead of a custom CA cert?